### ISSUE TITLE
Having a sliding list without options crashes

### ISSUE THREAD

**Issue by AmitMY** [driftyco/ionic#9914]
**Ionic version:**  (check one with "x")
[ ] **1.x**
[X] **2.x**

**I'm submitting a ...**  (check one with "x")
[X] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**
Having a sliding list without sliding crashes

**Expected behavior:**
Not to crash. Maybe just show a console error

**Steps to reproduce:**
http://plnkr.co/edit/v44PRUmIoRn87MWgJVY2?p=preview

**Related code:**

```
<ion-item-sliding>
      <ion-item>Slide me</ion-item>
</ion-item-sliding>
```

**Comment by jgw96** [comment]
Hello, thanks for using Ionic! Currently `ion-item-sliding` is meant to be used with `ion-item-options`, you can see an example of this in our [docs](http://ionicframework.com/docs/v2/components/#sliding-list). What is your use case for using `ion-item-sliding` without any options?

**Comment by AmitMY** [comment]
@jgw96 I am aware, thanks.
The use case:
For example, I have a chat app. The chat app has groups.
When you go in to group settings, as the group's manager, you want to be able to remove everyone, but yourself (that is the way whatsapp behaves, anyhow)
So I did a
`<ion-item-sliding *ngFor="let member of members">`
Then inside I did:
`<ion-item-options *ngIf="member is not group's manager">`

This caused a problem, when trying to slide the manager.


So basically when having a sliding list, with dynamic slide options.
